modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx

```diff
@@ -464,35 +529,38 @@ const FDSEntries = ({
 	];
 
 	return (
-		<>
-			<FrontendDataSet
-				apiURL={apiURL}
-				creationMenu={creationMenu}
-				customDataRenderers={{
-					provider: ProviderRenderer,
-				}}
-				id={`${namespace}FDSEntries`}
-				itemsActions={[
-					{
-						icon: 'view',
-						label: Liferay.Language.get('view'),
-						onClick: onViewClick,
-					},
-				]}
-				pagination={{
-					deltas: [
-						{label: 4},
-						{label: 8},
-						{label: 20},
-						{label: 40},
-						{label: 60},
-					],
-					initialDelta: 10,
-				}}
-				style="fluid"
-				views={views}
-			/>
-		</>
+		<FrontendDataSet
```

Comment:
I removed fragment in https://github.com/liferay-frontend/liferay-portal/pull/3113. There will be a minor conflict here, on PR that is merged second.